docs/superpowers/specs/2026-05-03-exercise-catalog-portal.md

```
@@ -0,0 +1,450 @@
+| BAR | Bar |
+| LONG_BAR | Long Bar |
+| SHORT_BAR | Short Bar |
+| ROPE | Rope |
+| BELT | Belt |
+| BENCH | Bench |
+| STRAPS | Straps |
+| GREY_CABLES | Cables |
+
+### Display Name Generation Rule
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)